llvm/lib/Target/AMDGPU/SIISelLowering.cpp

```diff
@@ -15931,6 +15931,16 @@ static OptimizationRemark emitAtomicRMWLegalRemark(const AtomicRMWInst *RMW) {
          << " operation at memory scope " << MemScope;
 }
 
+static bool isHalf2OrBFloat2(Type *Ty) {
+  if (FixedVectorType *VT = dyn_cast<FixedVectorType>(Ty)) {
```

Comment:
nit. I think LLVM standard suggests to use `auto` where cast is used.

```suggestion
  if (auto *VT = dyn_cast<FixedVectorType>(Ty)) {
```